Index: CLAPACK-3.1.1.1/F2CLIBS/libf2c/Makefile
===================================================================
--- CLAPACK-3.1.1.1.orig/F2CLIBS/libf2c/Makefile	2011-10-21 19:39:06.062426701 -0400
+++ CLAPACK-3.1.1.1/F2CLIBS/libf2c/Makefile	2011-10-21 19:42:29.992426572 -0400
@@ -19,8 +19,7 @@
 # compile, then strip unnecessary symbols
 .c.o:
 	$(CC) -c -DSkip_f2c_Undefs $(CFLAGS) $*.c
-	ld -r -x -o $*.xxx $*.o
-	mv $*.xxx $*.o
+
 ## Under Solaris (and other systems that do not understand ld -x),
 ## omit -x in the ld line above.
 ## If your system does not have the ld command, comment out
@@ -180,12 +179,6 @@
 xwsne.o:	lio.h
 xwsne.o:	fmt.h
 
-arith.h: arithchk.c
-	$(CC) $(CFLAGS) -DNO_FPINIT arithchk.c -lm ||\
-	 $(CC) -DNO_LONG_LONG $(CFLAGS) -DNO_FPINIT arithchk.c -lm
-	./a.out >arith.h
-	rm -f a.out arithchk.o
-
 check:
 	xsum Notice README abort_.c arithchk.c backspac.c c_abs.c c_cos.c \
 	c_div.c c_exp.c c_log.c c_sin.c c_sqrt.c cabs.c close.c comptry.bat \
